.github/actions/update-axe-core-v1/src/installScript.ts

```diff
@@ -1,26 +1,25 @@
 interface InstallScriptParams {
-  packageManager: string
+  packageManager: 'npm' | 'yarn'
   pinStrategy: string
   latestAxeCoreVersion: string
-  dependencyType: string
+  dependencyGroup: 'dependencies' | 'devDependencies'
 }
 
 export default function ({
   packageManager,
   pinStrategy,
   latestAxeCoreVersion,
-  dependencyType
+  dependencyGroup
 }: InstallScriptParams) {
-const newAxeVersion = `axe-core@${pinStrategy}${latestAxeCoreVersion}`
+  const newAxeVersion = `axe-core@${pinStrategy}${latestAxeCoreVersion}`
 
-  if (packageManager !== 'yarn') {
-    return ['i', newAxeVersion, dependencyType]
+  if (packageManager === 'npm') {
+    const dependencyTypeArgs = dependencyGroup === 'dependencies' ? [] : ['-D']
+    return ['i', ...dependencyTypeArgs, newAxeVersion]
+  } else if (packageManager === 'yarn') {
+    const dependencyTypeArgs = dependencyGroup === 'dependencies' ? [] : ['-D']
+    return ['add', ...dependencyTypeArgs, newAxeVersion]
```

Comment:
small nit (nothing blocking) - 

```suggestion
 const dependencyTypeArgs = dependencyGroup === 'dependencies' ? [] : ['-D']
  if (packageManager === 'npm') {
    return ['i', ...dependencyTypeArgs, newAxeVersion]
  } else if (packageManager === 'yarn') {
    return ['add', ...dependencyTypeArgs, newAxeVersion]
```